packages/mux-player/test/template.test.js

```diff
@@ -11,7 +11,7 @@ describe('<mux-player> template render', () => {
     render(content({}), div);
```

Comment:
non-blocking: If we end up settling on these slots, we should add more tests around how they behave under varying conditions